index.php

```diff
@@ -0,0 +1,161 @@
+<?php
+	require __DIR__ . '/vendor/autoload.php';
+
+	use Twig\Environment;
+	use Twig\Loader\FilesystemLoader;
+	
+	$loader = new FilesystemLoader(__DIR__ . '/templates');
+	$twig = new Environment($loader);
+	
+	session_start();
+
+    ini_set('display_errors', 1);
+    error_reporting(E_ALL);
+	require __DIR__ . '/vendor/autoload.php';
+	define("IN_INDEX", 1);
+
+    include("config.inc.php");
+
+    if (isset($config) && is_array($config)) {
```

Comment:
Reverse this if statement and then you don't need to indent try/catch block further.